src/pkg/control/restore.go

```diff
@@ -138,7 +138,7 @@ func (rc RestoreConfig) Conceal() string {
 // used within a PrintF, suitable for logging, storing in errors,
 // and other output.
 func (rc RestoreConfig) Format(fs fmt.State, _ rune) {
-	fmt.Fprint(fs, rc.concealed())
+	fmt.Fprint(fs, rc.concealed().marshal())
```

Comment:
To confirm - the issue was that `concealed()` returned a `RestoreConfig` which `Fprintf` would then call `Format` on?